_blog/2018/egress-tcp.md

```diff
@@ -257,6 +257,11 @@ Note that all the IPs of an external service are not always known. To enable TCP
 
 Also note that the IPs of an external service are not always static, for example in the case of [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network). Sometimes the IPs are static most of the time, but can be changed from time to time, for example due to infrastructure changes. In these cases, if the range of the possible IPs is known, you should specify the range by CIDR blocks, by multiple egress rules if needed, as in the case of `wikipedia.org`,  described in [Control Egress TCP Traffic Task]({{home}}/docs/tasks/traffic-management/egress-tcp.html). If the range of the possible IPs is not known, egress rules for TCP cannot be used and [the external services must be called directly]({{home}}/docs/tasks/traffic-management/egress.html#calling-external-services-directly), circumventing the sidecar proxies.
 
+## Relation to mesh expansion
+Note that the scenario described in this post is different from the mesh expansion scenario, described in the [Integrating Virtual Machines](https://istio.io/docs/guides/integrating-vms.html) guide. In that scenario, a MySQL instance runs on a an external (outside the cluster) machine (a bare metal or a VM), integrated with the Istio service mesh. The MySQL service becomes a first-class citizen of the mesh with all the beneficial features of Istio applicable. Among other things, the service becomes addressable by a local cluster domain name, for example by `mysqldb.vm.svc.cluster.local`. There is no need to create an egress rule to access this service, however the service has to be registered with Istio. To enable such integration, Istio components (_Envoy proxy_, _node-agent_, _istio-agent_) must be installed on the machine and the Istio control plane (_Pilot_, _Mixer_, _CA_) must be accessible from it. See the [Istio Mesh Expansion](https://istio.io/docs/setup/kubernetes/mesh-expansion.html) instructions for more details.
```

Comment:
Thanks for that. Can you add "including mTLS security" to the Among other things or benefits

Reply:
@ldemailly Done, thanks!